### PR TITLE
Office updates

### PR DESCRIPTION
**Pull request by diocas**
Giuseppe, can you have a look if everything is ok? (please ignore the ugliness of the code)

And can you do some tests in webng? They were deployed on Monday and during the tests we did today everything seemed to be fine...

### CODE REVIEW

**Comment by glpatcern** [comment]
I guess all this lives in `cbox-webng`, I'll go for some tests myself and then it's good to go (apart the minor question in the code)

File: ocproxy/api/api.go [hunk 1]
```diff
@@ -538,12 +597,30 @@ func (p *proxy) onlyOfficeTrackInternal(w http.ResponseWriter, r *http.Request,
...
-			return
+
+			if err != nil {
+				p.logger.Error(err.Error())
+
+				w.Header().Set("Content-Type", "application/json; charset=utf-8")
+				w.Write(encoded)
+				return
+
+			} else {
```
Comment (labkode) [inline ocproxy/api/api.go:612]: @diocas sleep 2 seconds here

**Comment by labkode** [comment]
@diocas just one comment and is good to go

**Comment by diocas** [comment]
Added the sleep + some log messages + stopped returning 500 to OO when something went bad with the locks

**Comment by labkode** [comment]
@diocas rebase with master and I merge

**Pull request merged**